src/ConnectedKubernetes/ConnectedKubernetes.Autorest/custom/New-AzConnectedKubernetes.ps1

```diff
@@ -599,9 +600,14 @@ function New-AzConnectedKubernetes {
 
         $PSBoundParameters.Add('ArcAgentryConfiguration', $arcAgentryConfigs)
 
-        Write-Verbose "Creating 'Kubernetes - Azure Arc' object in Azure"
+        Write-Host "Creating 'Kubernetes - Azure Arc' object in Azure"
         $Response = Az.ConnectedKubernetes.internal\New-AzConnectedKubernetes @PSBoundParameters
 
+        if ((-not $WhatIfPreference) -and (-not $Response)) {
+            Write-Error "Failed to create the connected cluster resource."
```

Comment:
Maybe change to "Failed to create the 'Kubernetes - Azure Arc' resource" as this seems to be the resource name shown in the portal.